data/repos.yml

```diff
@@ -1,3 +1,7 @@
+- repo_name: accessible-autocomplete-multiselect
+  type: Node module
```

Comment:
It looks like you're introducing a new repo type here – I'm not sure if that's supported, or if we have to stick to the existing types. Do you know?

It looks like 'utilities' is the usual dumping ground for miscellaneous repos – including `miller-columns-element`:

https://github.com/alphagov/govuk-developer-docs/blob/40411b6a840f59e27c8c96ccaa22d1699b25a4bc/data/repos.yml#L881-L883

I think we should either:

1. introduce a 'Node module' repo type, and recategorise other repos in the process (e.g. miller-columns-element)
2. categorise this new accessible-autocomplete-multiselect repo as a utility

What do you think @DilwoarH?

Reply:
I think to keep things simple; I'll just change it to utilities